### ISSUE TITLE
length of pluginname?

### ISSUE THREAD

**Issue by danmarsden** [marcusgreen/moodle-qbehaviour_interactiveexplain#5]
tablenames are restricted to 28 chars and must use the frankenstyle plugin name as a prefix.

If there's no chance this plugin will require it's own tables this is probably ok and you should close this as won't fix, but if you have plans that involve adding tables to the plugin you might want to rename it to something shorter.

**Comment by marcusgreen** [comment]
Unless Tim Hunt or someone in a similar role does something quite unexpected there will never be any database tables in this plugin. I am minded to close this as won't fix. However should I (can I ?) remove the whole db folder as it is never going to be used.

**Comment by danmarsden** [comment]
Yeah makes sense to remove it - afaik it isnt a required folder

**Comment by marcusgreen** [comment]
Have removed db folder will close ticket

**Comment by danmarsden** [comment]
not sure if you saw CONTRIB-8036 ? - just waiting on a response from you there before I continue any review - particularly with your naming of the matching settings plugin.
